File: sdks/go/pkg/models/shared/legalholdpolicies.go
package shared

// LegalHoldPolicies
// The part of an API response that describes marker
// based pagination
type LegalHoldPolicies struct {
	Entries    []LegalHoldPolicy `json:"entries,omitempty"`
	Limit      *int64            `json:"limit,omitempty"`
	NextMarker *int64            `json:"next_marker,omitempty"`
	PrevMarker *int64            `json:"prev_marker,omitempty"`
}

File: sdks/python/src/boxsdk/models/shared/legalholdpolicies.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from boxsdk import utils
from ..shared import legalholdpolicy as shared_legalholdpolicy


@dataclass_json
@dataclasses.dataclass
class LegalHoldPolicies:
    r"""LegalHoldPolicies
    The part of an API response that describes marker
    based pagination
    """
    
    entries: Optional[list[shared_legalholdpolicy.LegalHoldPolicy]] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('entries') }})
    limit: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('limit') }})
    next_marker: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('next_marker') }})
    prev_marker: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('prev_marker') }})
    
